tests/link/numba/test_compile_ops.py

```diff
@@ -153,3 +153,52 @@ def test_check_and_raise():
     out = assert_op(x.sum(), np.array(True))
 
     compare_numba_and_py([x], out, [x_test_value])
+
+
+def test_ifelse_single_output():
+    x = pt.vector("x")
+    out = ifelse(x.sum() > 0, x, x)
+
+    fn = function([x], out, mode=Mode("numba", optimizer=None))
+
+    x_test = np.zeros((5,))
+    res = fn(x_test)
+
+    # Returned array should not be the input (must be a copy)
+    assert res is not x_test
+    assert np.array_equal(res, x_test)
+
+
+def test_ifelse_multiple_outputs():
+    x = pt.vector("x")
+    y = pt.vector("y")
+    out1, out2 = ifelse(x.sum() > 0, (x, y), (y, x))
+
+    fn = function([x, y], [out1, out2], mode=Mode("numba", optimizer=None))
+
+    a = np.ones(3)
+    b = np.zeros(3)
+
+    r1, r2 = fn(a, b)
+
+    assert np.array_equal(r1, a)
+    assert np.array_equal(r2, b)
+    assert r1 is not a
+    assert r2 is not b
+
+
+def test_ifelse_false_branch():
```

Comment:
You can merge this test with the previous ones. Just eval the function twice, in a way that triggers the different branches.